lua/pac3/editor/client/saved_parts.lua

```diff
@@ -118,7 +118,7 @@ function pace.Backup(data, name)
 		end
 	end
 
-	if #data > 0 then
+	if data[1] then
```

Comment:
Should be `if data[1]~=nil then` in case the table contains falses.

Reply:
Good spot, I've applied your suggestion for all these 😄